lib/protocol.js

```diff
@@ -18,7 +18,8 @@ var ENCODINGS = [
   messages.Unpause,
   messages.Request,
   messages.Response,
-  messages.Cancel
+  messages.Cancel,
+  messages.Extension
```

Comment:
we shouldn't add a new message for extensions. if we use the approach described in https://github.com/mafintosh/hypercore/issues/4 all messages tagged >64 will be considered extension messages